Updated outdated references to 'documentdb'

mjalwajeeh-personal opened the pull request:
Replaced outdated references to 'documentdb' with 'cosmosdb' along with the correct link to the cosmosdb-create cli docs.

azurecla commented:
Hi __@stuartozer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stuarto). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Approved via mail from Stuart to me and Sidney.

stuartozer commented:
Made another to update the title!
 
Date: Mon, 1 Jun 2015 11:27:54 -0700
From: notifications@github.com
To: azure-content@noreply.github.com
CC: stuartozer@hotmail.com
Subject: Re: [azure-content] Update sql-database-elastic-query-overview.md (#3664)

Merged #3664.


—
Reply to this email directly or view it on GitHub.

Commit:
Updated outdated references to 'documentdb'

Replaced outdated references to 'documentdb' with 'cosmosdb' along with the correct link to the cosmosdb-create cli docs.
